pkg/controller/volume/pvcprotection/pvc_protection_controller.go

```diff
@@ -133,7 +149,7 @@ func (c *Controller) processNextWorkItem(ctx context.Context) bool {
 		return true
 	}
 
-	err = c.processPVC(ctx, pvcNamespace, pvcName)
+	err = c.processPVC(ctx, pvcKey, pvcNamespace, pvcName)
```

Comment:
How about just passing `pvcKey` (without `pvcNamespace` and `pvcName`) and moving `cache.SplitMetaNamespaceKey` to `c.processPVC`?

Reply:
Good point. But the return values of `processPVC` have to be changed to `(forget bool, err error)` if we move `cache.SplitMetaNamespaceKey` into it. Because it is expected not to forget the key if it fails to `cache.SplitMetaNamespaceKey` to avoid repeatedly processing the invalid key. WDYT?

Reply:
Ah, you are right. I've missed this. Let's keep it as is then.